packages/account-tree-controller/src/AccountTreeController.ts

```diff
@@ -462,6 +462,8 @@ export class AccountTreeController extends BaseController<
         }
       }
     });
+
+    this.messagingSystem.publish(`${controllerName}:accountTreeUpdated`);
```

Comment:
WDYT about also publishing `state.accountTree` here? That would be very similar to `:stateChange`?